### DISCUSSION TITLE
I didn't receive student developer pack offers (5d)

### DISCUSSION THREAD

**Discussion by shishkin-github** [community/community#112958] [GitHub Education] [answered]
### Hi everyone,

Hey everyone, i were last verified as a student on Mar 13, 2024. But it's already Mar 18, i didn't receive student developer pack offers. When i can get them? Thank you

**Comment by ghost** [comment]
Hi it usually takes time between 6 - 7 days after approval don't worry , 
Then if you had issues open a discussion about it or contact GitHub supports.

Please mark this discussion as answered if it was helpful
Good luck .

**Comment by shishkin-github** [comment, marked as answer]
Thank you for your answer

**Reply by ghost** [reply]
Anytime

Please mark this discussion as answered thank you.
